images/cassandra/rootfs/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/dumb-init /bin/bash
```

Comment:
Plus it leads to weird cases where Dockerfile has `dumb-init` running `dumb-init` running `bash`